tutorials/annotator.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Annotator
+---
+
+Syntax highlight on a complete AST is based on [`Annotator`](upsource:///platform/analysis-api/src/com/intellij/lang/annotation/Annotator.java),
```

Comment:
This topic doesn't appear to be saying anything new; why add it?

Reply:
Because it's a grey chapter. If you consider it unnecessary, we should remove it, both this file and the "Annotator" subsection in the index.

Reply:
What do you think?

Reply:
No, there is something to be written for this topic (because annotators make sense not only for custom language developers; there are usecases when an annotator for Java is useful), but I'd rather create a topic only when there is some actual content. Before that, it's OK to leave a placeholder in the TOC.

Reply:
Get.